ssm/transitions.py

```diff
@@ -264,6 +274,8 @@ def log_transition_matrices(self, data, input, mask, tag):
     def m_step(self, expectations, datas, inputs, masks, tags, **kwargs):
         Transitions.m_step(self, expectations, datas, inputs, masks, tags, **kwargs)
 
+    def hessian_expected_log_trans_prob(self, data, input, mask, tag, expected_joints):
```

Comment:
Here's where I would compute the Hessian manually.  This class is essentially a multiclass logistic regression.